apps/files_sharing/lib/Controller/ShareAPIController.php

```diff
@@ -1003,6 +1003,14 @@ public function getInheritedShares(string $path): DataResponse {
 		return new DataResponse(array_values($shares));
 	}
 
+	/**
+	 * Check wether a permissions set contains some permissions.
```

Comment:
```suggestion
	 * Check whether a set of permissions contains the permissions to check.
```
The other `wether` typos should also be fixed

Reply:
Not the first time :grimacing: 
Thanks!